vendor/engines/secure_rooms/app/views/secure_rooms/occupancies/index.html.haml

```diff
@@ -0,0 +1,24 @@
+= content_for :h1 do
+  = current_facility
+= content_for :sidebar do
+  = render "admin/shared/sidenav_product", sidenav_tab: "secure_rooms"
+= content_for :tabnav do
+  = render "admin/shared/tabnav_product", secondary_tab: "occupancies"
+
+%h2= @product
+
+%p= text("views.admin.secure_rooms.occupancies.index.description")
+
+- if @occupancies.empty?
```

Comment:
Not a big deal, especially as this is the more common pattern in NUcore, but I'm generally preferring these conditionals to be flipped so the positive case is first and the none case is at the bottom of the file.